glide.yaml

```diff
@@ -3,12 +3,17 @@ import:
 - package: github.com/cenkalti/backoff
   version: v1.0.0
 - package: github.com/giantswarm/microkit
+  version: master
   subpackages:
   - error
   - logger
 - package: github.com/juju/errgo
+  version: master
 - package: k8s.io/client-go
-  version: v3.0.0-beta.0
+  version: v3.0.0
+# apimachinery is pinned to the version client-go uses.
+- package: k8s.io/apimachinery
+  version: 85ace5365f33b16fc735c866a12e3c765b9700f2
```

Comment:
I also pinned apimachinery. `master` doesn't work with `v3.0.0`. It's better do to it here once than in every project using operatorkit.